### PR TITLE
Start a HOWTO about patch management

### PR DESCRIPTION
**Pull request by whit537**
Reeeeaallly basic

### CODE REVIEW

File: www/howto/patch-manage.spt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+nav_title = "Patch Manage"
```
Comment (nashe) [inline www/howto/patch-manage.spt]: Not "Patch Management"?

Comment (whit537) [inline www/howto/patch-manage.spt]: @nashe The pattern is "How to [do something]"

Comment (whit537) [inline www/howto/patch-manage.spt]: We could switch to "Manage Patches"?

Comment (nashe) [inline www/howto/patch-manage.spt]: Sounds better to my *english-is-not-my-native-language* ear :-)

Comment (whit537) [inline www/howto/patch-manage.spt]: Yeah, you're right, "patch manage" is too clever. :)

#720

**Pull request merged**